utils/src/main/proto/settings.proto

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2023 Cyface GmbH
+ *
+ * This file is part of the Cyface Utils for Android.
+ *
+ * The Cyface Utils for Android is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Cyface Utils for Android is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the Cyface Utils for Android. If not, see <http://www.gnu.org/licenses/>.
+ */
+syntax = "proto3";
+
+option java_package = "de.cyface.utils";
+option java_multiple_files = true;
+
+/**
+ * The data types for the values stored in the `AppSettings`.
+ *
+ * See https://protobuf.dev/programming-guides/proto3/
+ *
+ * Attention: The classes are generated from the file at compile time. Don't forget to rebuild.
+ *
+ * @author: Armin Schnabel
+ * @since: 4.0.0
+ * @version: 1.0.0
+ */
+message Settings {
+  // Version of this datastore. It helps to support default values and to migrate data. When the
+  // datastore is newly created, it's 0, after default values are set it's 1, then migration happens.
+  uint32 version = 1;
+
+  // Whether the map should be automatically centered while moving
+  bool center_map = 2;
+
+  // Whether the app should be able to upload measurements.
+  bool upload_enabled = 3;
+
+  // The maximum frequency with which the IMU sensors should collect data, e.g. 100 Hz.
+  int32 sensor_frequency = 4;
+
+  // Whether the app should report to the error monitoring service.
+  bool report_errors = 5;
+
+  // The currently selected modality, e.g. 'CAR'.
+  string modality = 6;
```

Comment:
could be an enum, but to keep the PRs as simple as possible (they are already big) I stayed with a string. We can add another migration step (version 2) later to do this.